.firebaserc

```diff
@@ -2,7 +2,7 @@
   "projects": {
     "dev": "community-map-dev",
     "master": "community-map-dev",
-    "production": "community-map-dev",
+    "prod": "communitymap42",
```

Comment:
@prescottprue this should match the name of the branch used for production and for deploy with _firebase-ci_ which is "prod", right?

Reply:
@dmitry-yudakov That is correct